api/CanvasRenderingContext2D.json

```diff
@@ -0,0 +1,3859 @@
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "drawFocusIfNeeded": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/drawFocusIfNeeded",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "29",
+                "notes": "In Firefox 28 this method was implemented as <code>drawSystemFocusRing</code>, but has been renamed to the standard <code>drawFocusIfNeeded</coded> in Firefox 29."
+              },
+              {
+                "version_added": "32",
+                "notes": "Prior to Firefox 32 this method was deactivated by default behind the flag <code>canvas.focusring.enabled</code>."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "29",
+                "notes": "In Firefox 28 this method was implemented as <code>drawSystemFocusRing</code>, but has been renamed to the standard <code>drawFocusIfNeeded</coded> in Firefox 29."
+              },
+              {
+                "version_added": "32",
+                "notes": "Prior to Firefox 32 this method was deactivated by default behind the flag <code>canvas.focusring.enabled</code>."
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "Path parameter": {
```

Comment:
Same here, no white space. Identifier could just be "path", and description "\<code>Path\</code> parameter".